src/ui/MessageContent/index.tsx

```diff
@@ -149,6 +158,11 @@ export default function MessageContent(props: MessageContentProps): ReactElement
   // Thread replies
   const displayThreadReplies = message?.threadInfo?.replyCount > 0 && replyType === 'THREAD';
 
+  // Feedback buttons
+  // TODO: Replace line below with comment before merging.
+  const isFeedbackMessage = !isByMe; // !isByMe && message?.myFeedbackStatus && message.myFeedbackStatus !== FeedbackStatus.NOT_APPLICABLE;
```

Comment:
Do we need to apply this todo?

Reply:
Thanks for the reminder ill put this back